pmd-apex/src/main/resources/category/apex/design.xml

```diff
@@ -488,4 +488,49 @@ public class Person {
         </example>
     </rule>
 
+
+    <rule name="AvoidBooleanMethodParameters"
```

Comment:
We try to sort the rule entries in the rulesets alphabetically by name. Can you add this at the correct place rather than at the end?